kube-runtime/src/controller/mod.rs

```diff
@@ -352,14 +361,16 @@ where
         self.reader.clone()
     }
 
-    /// Indicate child objets `K` owns and be notified when they change
+    /// Specify `Child` objects which `K` owns and should be watched
     ///
-    /// This type `Child` must have [`OwnerReference`] set to point back to `K`.
-    /// You can customize the parameters used by the underlying `watcher` if
-    /// only a subset of `Child` entries are required.
-    /// The `api` must have the correct scope (cluster/all namespaces, or namespaced)
+    /// Takes an [`Api`] object that determines how the `Controller` listens for changes to the `Child`.
+    /// All owned `Child` objects **must** contain an [`OwnerReference`] pointing back to a `K`.
     ///
-    /// [`OwnerReference`]: https://docs.rs/k8s-openapi/0.10.0/k8s_openapi/apimachinery/pkg/apis/meta/v1/struct.OwnerReference.html
+    /// The [`ListParams`] refer to the possible subset of `Child` objects that you want to own and
+    /// and receive reconcile events for.
+    /// To own the full set of `Child` objects, you can use `ListParams::default`.
```

Comment:
Ownership is controlled by `OwnerReference`s, this is for filtering out a subset of those owned resources.

Reply:
swapping own uses here for track and watch